linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/H2AccessLogger.scala

```diff
@@ -0,0 +1,83 @@
+package io.buoyant.linkerd.protocol.h2
+
+import com.twitter.finagle.{Service, ServiceFactory, SimpleFilter, Stack, Stackable}
+import com.twitter.finagle.buoyant.h2.{Request, Response}
+import com.twitter.logging._
+import com.twitter.util.{Time, TimeFormat}
+
+case class H2AccessLogger(log: Logger) extends SimpleFilter[Request, Response] {
+
+  def apply(req: Request, svc: Service[Request, Response]) = {
+    val reqHeaders = req.headers
+    val remoteHost = "-"
```

Comment:
I think it would be good to log something for the `remoteHost` field, but as far as I can tell, this would probably require changes in `h2.Request` to store the remote address like Finagle's HTTP `Request` does, so it's reasonable to punt on that for now.

Reply:
yes, after discussing this with kevin he said to not change the Request object for now, so i'll leave this as is.

Reply:
I think you can get the remote host with this:

```
import com.twitter.finagle.context.RemoteInfo
RemoteInfo.Upstream.addr
```

Reply:
awesome! i'll give this a shot - thanks!

Reply:
nice, i didn't know that existed!